apps/urbit-bitcoin-connector/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  web:
+    image: matwet/urbit-btc-node:latest
```

Comment:
This Docker image is only built for amd64. Umbrel apps need to use multiarch containers that support both amd64 and arm64. Most of Umbrel (>99%) is running on arm64.

It's pretty simple to support with `docker buildx`, we have an example here: https://github.com/getumbrel/umbrel/blob/master/apps/README.md#a-good-dockerfile